exchange/exchange.go

```diff
@@ -202,6 +204,7 @@ func (e *exchange) HoldAuction(ctx context.Context, r AuctionRequest, debugLog *
 	if err != nil {
 		return nil, err
 	}
+	requestExt.Prebid.Server = &openrtb_ext.ExtRequestPrebidServer{ExternalUrl: e.server.ExternalUrl, GvlID: e.server.GvlID, Datacenter: e.server.Datacenter}
```

Comment:
Should we only be setting this if we have at least one piece of server data? Otherwise we end up with an empty `prebid.server` object in our bid requests.

Reply:
Probably a good idea. An `Empty` method on `ExtRequestPrebidServer` might be nice to check for this condition.

Reply:
That's a good check to add! Will update